README.md

```diff
@@ -126,3 +126,19 @@ Create a Github personal access token with repo scope and do:
 echo -e "machine raw.githubusercontent.com\nlogin <TOKEN>" | sudo tee /etc/apt/auth.conf.d/github.conf
 
 ```
+
+### How to use a private repo as a rosdep source
+
+Rosdep doesn't support authentication with private repose, so as a workaround you'll have to manually download the `local.yaml` file and reference that file in rosdep sources instead of the github url.
+
+```bash
+# Download the local.yaml file to path of your choice
+curl -H "Authorization: token <your_PAT_token>" -L https://raw.githubusercontent.com/<user>/<repo>/<branch>/local.yaml > local.yaml
+
+# Add the local.yaml file to rosdep sources instead of the github url
+echo "yaml file://<absolute_path_to_local.yaml> <ros_distro>" | sudo tee /etc/ros/rosdep/sources.list.d/1-<repo_name>.list
+```
+
+For example: `yaml file:///home/user/rosdep/local.yaml rolling`
```

Comment:
I think the file should be somewhere in `/etc/ros/rosdep`, referencing a file in `/home` from `/etc` looks wrong. Also maybe something different then `local.yaml`?